lib/model/bloc/plot/PlotStatistics.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:farmsmart_flutter/model/model/PlotEntity.dart';
+
+import 'StageBusinessLogic.dart';
+
+class PlotStatistics {
+
+  StageBusinessLogic _stageLogic = StageBusinessLogic();
+
+  int compeletedCount(List<PlotEntity> plots) {
+    int completed = 0;
+    plots.forEach((plot){
+      completed += _stageLogic.isComplete(plot.stages.last)? 1:0;
```

Comment:
Magic number
```suggestion
      completed += _stageLogic.isComplete(plot.stages.last)? 1 : 0;
```

Reply:
I think this is ok here and its a literal constant of 1. we're counting..

Reply:
👌